xml/Microsoft.Vsa/VsaLoader.xml

```diff
@@ -610,8 +610,8 @@
         <Parameter Name="PDB" Type="System.Byte[]" RefType="out" />
       </Parameters>
       <Docs>
-        <param name="PE">To be added.</param>
-        <param name="PDB">To be added.</param>
+        <param name="PE">The file formatted in PE.</param>
+        <param name="PDB">The file formatted in PDB.</param>
         <summary>This member supports the .NET Framework infrastructure and is not intended to be used directly from your code.</summary>
```

Comment:
We can't leave these "This member supports the .NET Framework infrastructure and is not intended to be used directly from your code." when doing these APIs. Add a summary and then add the forInternalUseOnly tag after remarks.